openshift/ocp_on_openstack.adoc

```diff
@@ -89,6 +96,176 @@ Push the `pass` changes to the git repository.
 git push
 ----
 
+== Post installation steps
+
+=== Install jump server
+
+The OCP cluster is created on a private network that allows communication 
+ between the different hosts of the cluster. To be able to SSH to any of 
+ the hosts of this private network a _jump server_ host is required. 
+ This host must be created inside the network of the OCP cluster.
+ 
+To access the _jump server_ a Public IP is required which can be 
+ accomplished by creating a RHOS _Floating IP_ and associating it with that 
+ host.
+
+The OCP private network is created automatically during the installation 
+ process. The name of this network is built using the `clusterName` defined 
+ in the installation process, a `<code>` that is a dynamic string generated by the installation process 
+ and the `openshift` suffix (`<clustername>-<code>-openshift`).
+
+==== Requirements
+
+*Collect the metadata of the installation executed previously.*
+
+The metadata of the installation can be obtained from the `metadata.json` 
+ file of the OCP installation directory.
+
+.Sample metadata.json
+[source,json]
+----
+{
+  "clusterName": "ocp",
+  "clusterID": "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx",
+  "infraID": "ocp-sscpc",
+  "openstack": {
+    "cloud": "openstack",
+    "identifier": {
+      "openshiftClusterID": "ocp-sscpc"
+    }
+  }
+}
+----
+
+[NOTE]
+====
+For the sake of example the `ocp-sscpc` prefix will be used.
+
+Also, the calculated RHOS internal network name should be  
+ `ocp-sscpc-openshift`.
+====
+
+==== Create and configure the server
+
+[WARNING]
+====
+Since each RHOS OCP cluster installation generates it's own RHOS network,  one _jump server_ will be required for each cluster. 
+ As a consequence the name of the _jump server_ 
+ should include the prefix of the cluster (`<clusterName>-<code>`) as identified in the metadata of the OCP cluster.
+====
+
+[source,bash]
+----
+ansible-playbook ansible/playbook/openstack/openstack_vm_create_passwordstore.yml \
+  -e '{"openstack": {"vm": {"network": "ocp-sscpc-openshift","image": "Fedora-Cloud-Base-29", "flavor": "m1.small"}}}' \
+  -e vm_name=ocp-sscpc-jump-server
+----
+
+Once the _jump server_ host is provisioned on RHOS, create a _Floating IP_ 
+ to be able to access that host. This _Floating IP_ must be associated 
+ with the RHOS _External network_ use by the OCP cluster hosts. This 
+ _External network_ can be identified from the RHOS _Router_ genereated 
+ by the RHOS OCP installation.
+ 
+To collect the name of the _External Network_ go to the
+ `Network > Routers` option of the RHOS console. 
+ The RHOS router for the OCP cluster can be identified by it's name 
+ that should be `<clusterName>-<code>-external-router`.
+ Once the router is identified check the `External Network` column, which in this example would be `ocp-sscpc-external-router`.
+
+The `ocp_cluster` Ansible Role as this value predefined in the installation 
```

Comment:
This text is not clear "The `ocp_cluster` Ansible Role as this value predefined in the installation 
procedure and it's `provider_net_cci_13`"

Reply:
Improved the text to try making it clearer.